charts/substra-backend/templates/deployment-server.yaml

```diff
@@ -135,19 +135,17 @@ spec:
           value: {{ .Values.postgresql.postgresqlPassword }}
         - name: DATABASE_HOST
           value: {{ .Release.Name }}-postgresql
-      {{- if eq .Values.backend.settings "prod" }}
```

Comment:
Could you please add context regarding this change?

Reply:
Because, the new way we serve the server needs the statics files :)